File: src/main/java/ru/ssau/tourism/entities/Voucher.java
package ru.ssau.tourism.entities;

import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;

@Entity
@Setter
@NoArgsConstructor
@EqualsAndHashCode(of = {"id"})
public class Voucher {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long id;
    @ManyToOne
    @JoinColumn(name = "tourist_id")
    private Tourist tourist;
    @ManyToOne
    @JoinColumn(name = "season_id")
    private Season season;

}

File: src/main/kotlin/ru/ssau/tourism/entities/Voucher.kt
package ru.ssau.tourism.entities

import javax.persistence.*

@Entity
class Voucher {
	@Id
	@GeneratedValue(strategy = GenerationType.AUTO)
	var id: Long? = null
	@ManyToOne
	@JoinColumn(name = "tourist_id")
	var tourist: Tourist? = null
	@ManyToOne
	@JoinColumn(name = "season_id")
	var season: Season? = null
}